File: specification/specs/analysis/tokenizers/PathHierarchyTokenizer.ts
class PathHierarchyTokenizer extends TokenizerBase {
	@prop_serializer("NullableStringIntFormatter")
	buffer_size: integer;
	delimiter: string;
	replacement: string;
	@prop_serializer("NullableStringBooleanFormatter")
	reverse: boolean;
	@prop_serializer("NullableStringIntFormatter")
	skip: integer;
}

File: output/csharp/Analysis/Tokenizers/PathHierarchyTokenizer.cs
using Nest.Internal;
using Nest.Analysis;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Analysis {

	public class PathHierarchyTokenizer : TokenizerBase {
		
		[DataMember(Name="buffer_size")]
		public int BufferSize { get; set; }


		[DataMember(Name="delimiter")]
		public string Delimiter { get; set; }


		[DataMember(Name="replacement")]
		public string Replacement { get; set; }


		[DataMember(Name="reverse")]
		public bool Reverse { get; set; }


		[DataMember(Name="skip")]
		public int Skip { get; set; }

	}
}
